src/Tabs/Tab.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+import { bemClassNames } from '../utils';
+import './tab.css';
+
+const bem = bemClassNames('d2ui-tab');
+
+const Tab = ({
+    icon,
+    label,
+    selectHandler,
+    active,
+    disabled,
+    stacked,
+    addTabRef,
+    children,
+}) => (
+    <button
+        className={`${bem.b({ active, disabled, stacked })} d2ui-align-icon`}
+        onClick={selectHandler}
+        ref={addTabRef}
+    >
+        {children
+            ? children
+            : (icon && <Icon name={icon} />, label && <span>{label}</span>)}
+    </button>
+);
+
+Tab.propTypes = {
+    label: PropTypes.string,
+    icon: PropTypes.string,
+    selectHandler: PropTypes.func,
+    addTabRef: PropTypes.func,
```

Comment:
This is interesting.... I had marked this prop as required, because it is actually needed for the component to function properly. However, this property is added by the `Tabs` component when it renders a `Tab`. So initially it isn't present. If I kept this property as required, it would keep throwing errors so I removed the `.isRequired` for now.....